contracts/script/deploy/router/CertVerifierRouterDeployer.s.sol

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.8.12;
+
+import {EigenDACertVerifierRouter} from "src/periphery/cert/router/EigenDACertVerifierRouter.sol";
+import "forge-std/Test.sol";
+import "forge-std/Script.sol";
+import "forge-std/StdJson.sol";
+
+/**
+ * @notice Deployment script for immutable EigenDACertVerifierRouter
+ * @dev Run with:
+ * forge script script/deploy/router/CertVerifierRouterDeployer.s.sol:CertVerifierRouterDeployer --sig "run(string, string)" <config.json> <output.json> --rpc-url $RPC --private-key $PRIVATE_KEY -vvvv --etherscan-api-key $ETHERSCAN_API_KEY --verify --broadcast
+ */
+contract CertVerifierRouterDeployer is Script, Test {
+    address eigenDACertVerifierRouter;
+    address initialOwner;
+
+    function run(string memory inputJSONFile, string memory outputJSONFile) external {
+        string memory path = string.concat("./script/deploy/router/config/", inputJSONFile);
+        string memory data = vm.readFile(path);
+
+        bytes memory raw = stdJson.parseRaw(data, ".initialOwner");
+        initialOwner = abi.decode(raw, (address));
+        raw = stdJson.parseRaw(data, ".initialCertVerifier"); // set at block height 0
```

Comment:
I think we wanted to move to using tomls instead of jsons. https://github.com/Layr-Labs/eigenda/tree/master/contracts/script/deploy/verifiable

Reply:
does toml make sense for all deployment types and ones where we have few fields?

Reply:
add ticket to move everything to toml configs